src/imgtools/dicom/input/sr_utils.py

```
@@ -0,0 +1,94 @@
+    a MR, we also get the SOP Instance UIDs
+
+    """
+    sr = load_dicom(sr)
+    if not "CurrentRequestedProcedureEvidenceSequence" in sr:
+        return None
+
+    series_uids = set()
+    sop_uids = set()
+
+    for evidence_seq in sr.CurrentRequestedProcedureEvidenceSequence:
+        if not "ReferencedSeriesSequence" in evidence_seq:
```

Ruff (0.8.2)

Test for membership should be `not in`

Convert to `not in`

(E713)